qir/qat/CMakeLists.txt

```
@@ -1,4 +1,6 @@
 cmake_minimum_required(VERSION 3.4.3)
+include(${CURRENT_SOURCE_DIR}version.cmake)
```

I can be wrong but I would expect something like 
```suggestion
include(${CURRENT_SOURCE_DIR}/version.cmake)
```

---

I've just doubled checked: It does not work if there is a slash.